WordPress/Classes/Plugins/Views/InstalledPluginsListView.swift

```diff
@@ -76,6 +73,9 @@ struct InstalledPluginsListView: View {
         .task {
             await viewModel.onAppear()
         }
+        .task(id: 1) {
```

Comment:
(nit) `id` is redundant

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/24097/commits/08b6c73fc012224bc854e7999aa453470445473b